tests/Intl/Idn/IdnTest.php

```diff
@@ -288,6 +288,15 @@ public function testEncodePhp53($decoded, $encoded)
         $this->assertSame($encoded, $result);
     }
 
+    /**
+     * @dataProvider domainNamesProvider
+     */
+    public function testEncodeDefaultPhp73($decoded, $encoded)
```

Comment:
this test does not make sense. It is not relying on a default encoding, and is overriding a PHP version check which does not make sense either.

Reply:
Problem related to php version checking, encode/decode not working with PHP 7.3 only. Of course i can rewrite it for only for check one item.

Reply:
the right way to check that the polyfill works on PHP 7.3 is to run the testsuite on PHP 7.3, not to fake a version check (which does not guarantee anything)